File: PatoEcommerceApp/src/Objects/Produto/ProductBrand.ts
import { ProductCategory } from "./ProductCategory"

export class ProductBrand {

    constructor(brandId: number, category: ProductCategory, name: string, description: string){
        this.BrandId = brandId;
        this.Category = category;
        this.Name = name;
        this.Description = description;
    } 

    BrandId: number = 0;

    Category: ProductCategory = new ProductCategory;

    Name: string = "";

    Description: string = "";
}

File: PatoEcommerce/EcommerceDLL/product/Productbrand.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace OpenEcommerceDLL.product
{
    public class ProductBrand
    {
        public int BrandId { get; set; }
        public ProductCategory? CategoryId { get; set; }
        public string? Name { get; set; }
        public string? Description { get; set; }

    }
}
